code/game/machinery/gulag_teleporter.dm

```diff
@@ -21,9 +21,17 @@ The console is located at computer/gulag_teleporter.dm
 	var/jumpsuit_type = /obj/item/clothing/under/rank/prisoner
 	var/shoes_type = /obj/item/clothing/shoes/sneakers/orange
 	var/obj/machinery/gulag_item_reclaimer/linked_reclaimer = null
+	var/list/required_items
 
 /obj/machinery/gulag_teleporter/New()
 	..()
+	required_items = typecacheof(list(
+				/obj/item/weapon/implant,
+				/obj/item/clothing/suit/space/eva/plasmaman,
+				/obj/item/clothing/under/plasmaman,
```

Comment:
I kept forgetting about this object for like 30 minuites. Literally the reason why I took so long to get it sorted today.